internal/service/streaminstance/data_source_stream_instances_test.go

```diff
@@ -23,8 +21,8 @@ func TestAccStreamDSStreamInstances_basic(t *testing.T) {
 		CheckDestroy:             acc.CheckDestroyStreamInstance,
 		Steps: []resource.TestStep{
 			{
-				Config: streamInstancesDataSourceConfig(orgID, projectName, instanceName, region, cloudProvider),
-				Check:  streamInstancesAttributeChecks(dataSourceName, nil, nil, 1),
+				Config: streamInstancesDataSourceConfig(projectID, instanceName, region, cloudProvider),
+				Check:  streamInstancesAttributeChecks(dataSourceName, nil, nil, acc.IntGreatThan(0)), // at least on instance is present in the project
```

Comment:
can we change this and check something that uniquely identifies the instance?

Reply:
good idea, apart from checks results is larger than 0 added a `TestCheckTypeSetElemNestedAttrs` to verify it contains an element with the stream instance name that was created in the test.